modules/Books.js

```diff
@@ -0,0 +1,90 @@
+const listBooks = document.querySelector('.bookList');
+const form = document.querySelector('.formInput');
+const [title, author] = form.elements;
+
+const inputBook = {};
+// eslint-disable-next-line import/no-mutable-exports
```

Comment:
- [ ] Kindly remove the `eslint-disable` and fix the linters. The linters are there for a purpose of preventing bugs and they should not be avoided.